gcc/config/riscv/riscv-vector-builtins-bases.cc

```diff
@@ -705,6 +705,31 @@ class vnclip : public function_base
   }
 };
 
+/* Implements vfnrclip.  */
+class vfnrclip : public function_base
+{
+public:
+  bool has_merge_operand_p () const override { return false; }
+  bool may_require_qfrm_p () const override { return true; }
+  bool can_be_overloaded_p (enum predication_type_index pred) const override
+  {
+    if (pred == PRED_TYPE_mu || pred == PRED_TYPE_tu || pred == PRED_TYPE_tumu)
+    {
+      return true;
+    }
+  }
+
+  rtx expand (function_expander &e) const override
+  {
+    if (e.op_info->op == OP_TYPE_f_qf)
+    {
+      return e.use_exact_insn (
+	  code_for_pred_fnr_clip (ZERO_EXTEND, e.vector_mode ()));
+      gcc_unreachable ();
+    }
+  }
```

Comment:
```suggestion
    }
    gcc_unreachable ();
  }
```
gcc_unreachable in wrong level I think?